app/src/main/java/com/deframe/artapp/ui/MapViewFragment.kt

```diff
@@ -71,9 +71,12 @@ class MapViewFragment : Fragment(), OnMapReadyCallback {
         }
 
         //runs a thread to call the database
-        Thread(Runnable {
+        var t = Thread(Runnable {
```

Comment:
What is the reason of this change? Is it related to the MapView Bug?

Reply:
Yes, please try it out and see if it still causes the bug. I cannot test on my side due to different thread running time. Thanks!

Reply:
Sure.